### PR TITLE
Add devicetime unittest

### PR DESCRIPTION
**Pull request by ki4070ma**
https://github.com/appium/python-client/issues/283

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Thanks. The method can set `format` argument like below.
https://github.com/appium/ruby_lib_core/blob/680577d83356fc19d5fc435d5e49f0fea3c1442a/lib/appium_lib_core/common/base/driver.rb#L613-L627

Can you extend current method format to it if you have a chance?

**Comment by ki4070ma** [comment]
Thanks for your review.
And ok, will do.

File: test/unit/webdriver/device/device_time_test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    @httpretty.activate
+    def test_device_time(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.GET,
+            appium_command('/session/1234567890/appium/device/system_time'),
+            body='{"value": ""}'
+        )
+        driver.device_time  # Expected no exception
```
Comment (KazuCocoa) [inline test/unit/webdriver/device/device_time_test.py]: Can you add one assertion?

Comment (ki4070ma) [reply test/unit/webdriver/device/device_time_test.py]: Thanks, my assertion was not enough. 
I've changed here to check if return value is same to value in body.

**Pull request merged**